package.json

```diff
@@ -18,70 +24,71 @@
 		"labels:dry": "github-label-sync --labels ./.github/label-sync-config.json --allow-added-labels --dry-run woocommerce/woocommerce-admin",
 		"labels:sync": "github-label-sync --labels ./.github/label-sync-config.json --allow-added-labels woocommerce/woocommerce-admin",
-		"lint": "npm run lint:js && npm run lint:css",
+		"lint": "yarn run lint:js && yarn run lint:css",
 		"lint:css": "stylelint '**/*.scss'",
 		"lint:css-fix": "stylelint '**/*.scss' --fix --ip 'storybook/wordpress'",
 		"lint:js": "wp-scripts lint-js ./packages ./client --ext=js,ts,tsx",
 		"lint:js:packages": "wp-scripts lint-js ./packages --ext=js,ts,tsx",
 		"lint:js:client": "wp-scripts lint-js ./client --ext=js,ts,tsx",
-		"lint:js-fix": "npm run lint:js -- --fix --ext=js,ts,tsx",
+		"lint:js-fix": "yarn run lint:js --fix --ext=js,ts,tsx",
 		"lint:php": "./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
 		"lint:php-fix": "./vendor/bin/phpcbf --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
 		"ts:check": "tsc --build",
 		"reformat-files": "wp-scripts format-js -- --ignore-path .eslintignore",
-		"prepack": "npm install && npm run lint && npm run test && cross-env WC_ADMIN_PHASE=core npm run build",
-		"publish-packages:check": "npm run build:packages && lerna updated",
-		"publish-packages:dev": "npm run build:packages && lerna publish from-package --npm-tag next",
-		"publish-packages:prod": "npm run build:packages && lerna publish from-package",
-		"prestart": "npm run -s install-if-deps-outdated",
-		"start": "cross-env WC_ADMIN_PHASE=development npm run build:packages && cross-env WC_ADMIN_PHASE=development npm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"node ./bin/packages/watch.js\"",
-		"pretest": "npm run -s install-if-no-packages",
-		"test": "./node_modules/jest-24.9.0/bin/jest.js --config tests/js/jest.config.json",
-		"test:debug": "node --inspect-brk ./node_modules/jest-24.9.0/bin/jest.js --config tests/js/jest.config.json --watch --runInBand --no-cache",
-		"test:e2e": "WP_VERSION=latest npm run build && npx wc-e2e docker:down && npx wc-e2e docker:up && npx wc-e2e test:e2e",
-		"test-staged": "./node_modules/jest-24.9.0/bin/jest.js --bail --config tests/js/jest.config.json --findRelatedTests",
+		"prepack": "yarn install && yarn run lint && yarn run test && cross-env WC_ADMIN_PHASE=core yarn run build",
+		"publish-packages:check": "yarn run build:packages && lerna updated",
+		"publish-packages:dev": "yarn run build:packages && lerna publish from-package --npm-tag next",
+		"publish-packages:prod": "yarn run build:packages && lerna publish from-package",
+		"prestart": "yarn run -s install-if-deps-outdated",
+		"start": "cross-env WC_ADMIN_PHASE=development yarn run build:packages && cross-env WC_ADMIN_PHASE=development yarn run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"yarn node ./bin/packages/watch.js\"",
```

Comment:
I'm getting an error on this command:

```
"yarn node ./bin/packages/watch.js\"
```

<img width="775" alt="Screen Shot 2021-07-06 at 12 13 14 PM" src="https://user-images.githubusercontent.com/1922453/124525490-98138500-de53-11eb-904a-27a8b9877a69.png">

As a result, `yarn start` works but it doesn't watch the packages. It only builds them once. If I `cd packages/components` and `yarn start` at the same it, that works.

Reply:
`yarn node ./bin/packages/watch.js packages/components` doesn't pick up changes either. 

I think `bin/packages/watch.js` could handle building all the packages if no path is given. It appears to only be used by the components package for building CSS, but thats not working.